_posts/blog/2019-07-18-lab-meeting.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Lab Meeting on 18 July 2019 ### REPLACE `DATE` with the date, eg: 18 July 2019
+categories: blog
+excerpt: Notes from the group's weekly lab meeting ### ADD a short description (or keep that one if you'd)
+tags: [lab-meeting] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link: 
+date: 2019-07-18 ### UPDATE date
+modified:
+share: true
+author: kirstie_whitaker ### CHANGE to your author name (in _data/authors.yml)
+---
+
+This week we had a guest speaker - [Bastian Greshake Tzvoras](#visiting-speaker-bastian-greshake-tzvoras) - but still managed to zoom through our celebrations and cool things to share.
+
+We asked, but didn't really answer, a few questions!
+Everyone (not just lab members) is welcome to brainstorm together in in our [gitter room](https://gitter.im/WhitakerLab/Lobby).
+
+## Celebrations and cool things to share
+
+**Kirstie** wrote a [little blog post](https://news.sky.com/story/turings-story-is-motivation-to-keep-fighting-for-equity-and-justice-11763751) for Sky News about Alan Turing being on the £50 note.
+
+The piece closes with the following call to action:
+
+> Alan Turing will now become the iconic face of the £50 note.
+> But he also represents everyone who has served their country, those giants of science on whose shoulders we now stand, and all people who have been - and continue to be - denied their human rights.
+> 
+> I look to Alan Turing both as a role model to inspire new and creative research questions and as motivation to keep fighting for equity and justice for all marginalised members of our society.
+> In his own words: **"We can only see a short distance ahead, but we can see plenty there that needs to be done."**
+
+**Sarah's** got herself into an infinite loop of [building cool bots](https://github.com/HelmUpgradeBot?tab=repositories) to do her work for her 😂
+
+**Patricia** is wrapping up her current job and getting ready for new adventures.
+But before she goes she'll run a "Friendly GitHub" workshop (inspired by [this one](https://kirstiejane.github.io/friendly-github-intro/)) for library colleagues next week.
+
+**Louise** has been learning about scientific file formats and how to improve them by learning from database design expertise.
+Kirstie recommended that Louise (and everyone!) should checkout Saul Pwanson's amazing talk at [csv,conf,v4](https://csvconf.com/) on ["How a File Format Led to a Crossword Scandal"](https://www.youtube.com/watch?v=9aHfK8EUIzg).
+You can see all the csv,conf talks online at this [YouTube playlist](https://www.youtube.com/playlist?list=PLg5zZXwt2ZW72M0KOOt8xQNXsSPMp4PNd).
+
+**Georgia** as built a demo for the Autistica/Citizen science platform!
```

Comment:
Tiny Typo to fix - "as" to "has"

Reply:
Thank you @GeorgiaHCA!! Fixed now.

Please can you re-review and approve when you have time 😄